### PR TITLE
internal/router: Less verbose tracking on some meta endpoints

### PR DESCRIPTION
**Pull request by mhemmings**
Less verbose tracking on some meta endpoints, hiding entity specific things in URL

### CODE REVIEW

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-gce.jujugui.org:8080//job/charmstore/2284/

**Review by mhilton — APPROVED** [review]
LGTM

File: internal/router/router.go [hunk 1]
```diff
@@ -347,7 +347,23 @@ func (r *Router) serveIds(w http.ResponseWriter, req *http.Request) error {
...
 	}
-	r.Monitor.SetEndpoint("/:id/meta" + path)
+
+	// We don't want to track things that are unique to entities.
+	// For example:
+	//     /wordpress/meta/resources/foo
+	//     We want to track as /:id/meta/resources/
+	//   and
+	//     /wordpress/meta/resources
+	//     We want to track as /:id/meta/resources
```
Comment (rogpeppe) [inline internal/router/router.go:357]: How about this? (the comment being added to your existing text above)

     // Technically some things after the slash might not be unique
     // to entities (for example meta/perm/read) but it's a good enough approximation
     // for now.

     if i := strings.Index(path+1, "/"); i > 0 {
         monitorPath = path[0:i+2]
     }

**Review by rogpeppe — APPROVED** [review]
LGTM with one suggestion, thanks.

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-gce.jujugui.org:8080//job/charmstore/2285/

**Pull request merged**